src/Smidge.Core/SmidgeConfig.cs

```diff
@@ -34,6 +34,8 @@ public SmidgeConfig()
         public string Version => _config["version"] ?? "1";
 
         public string DataFolder => (_config["dataFolder"] ?? "Smidge").Replace('/', Path.DirectorySeparatorChar);
+
+		public bool ProtectFileExtensions => bool.Parse((_config["protectFileExtensions"] ?? "false"));
```

Comment:
This name is odd to me as there's nothing to protect. I think KeepFileExtensions would be a better name.

Reply:
@PhyxionNL I've renamed everything to `keepFileExtensions`, including the 2 new tests as `Keep_File_Extensions`.

I noticed the tabs vs.spaces on line 38. I'm going to save checking this in until I have addressed your related feedback. 

Thank you!